tinkar-maven-plugin/src/main/java/dev/ikm/maven/DatastoreProxy.java

```diff
@@ -14,7 +14,7 @@ public class DatastoreProxy implements Closeable {
 
 
     public void start() {
-        File datastoreDirectory = new File(System.getProperty("project.build.directory"), "datastore");
```

Comment:
For now this works, but we should do things the "maven way" which would be to write to a database within the /target directory of a maven project. We can't assume there is always a ~/Solor directory